doc/src/modules/physics/mechanics/symsystem.rst

```diff
@@ -0,0 +1,184 @@
+=====================================
+Symbolic Systems in Physics/Mechanics
+=====================================
+
+The `SymbolicSystem` class in physics/mechanics is a location for the pertinent
+of a dynamic system. In its most basic form it contains the equations of motion
+for the dynamic system, however, it can also contain information regarding the
+loads that the system is feeling, the bodies that the system is comprised of
+and any additional equations the user feels is important for the system. The
+goal of this class is to provide a unified output format for the equations of
+motion that simulation code can be designed around.
+
+SymbolicSystem Example Usage
+============================
+
+This code will go over the manual input of the equations of motion for the
+simple pendulum into `SymbolicSystem` using x and y coordinates instead of
+theta.
+
+The equations of motion are formed at
+http://nbviewer.jupyter.org/github/bmcage/odes/blob/master/docs/ipython/Planar%20Pendulum%20as%20DAE.ipynb ::
```

Comment:
I found a page in the documentation of sympy mechanics that forms these equations of motion. The example though uses a different orientation of x, y axes than I do and uses q1 and q2 instead of x and y. I put a link to this example (I think, I'm better at working with markdown than rst) and mentioned the formation differences.